### PR TITLE
missing class from the tutorial example

### PR DESCRIPTION
**Pull request by stampycode**
Difference between "We will be styling this page"
at https://github.com/codebar/tutorials/blob/master/html/lesson2/tutorial.md#today-we-will-be-focusing-on-fundamental-css-concepts
and "Download the files required to begin working through the tutorial from here(mac/linux) or here(windows)"
at https://gist.github.com/hundred/7332441/download

### CODE REVIEW

File: html/lesson2/example.html [hunk 1]
```diff
@@ -45,10 +45,10 @@ <h3>Owls:</h3>
         </div>
         <ul id="links">
         <li>
-           <a href="mailto:social@codebar.io?subject=I love owls :: codebar">Email us</a>
+           <a href="mailto:social@codebar.io?subject=I love owls :: Codebar">Email us</a>
```
Comment (KimberleyCook) [inline html/lesson2/example.html]: sorry to be pedantic but codebar is always a lowercase c, so this is deliberate :)

File: html/lesson2/example.html [hunk 2]
```diff
@@ -45,10 +45,10 @@ <h3>Owls:</h3>
...
         </li>
         <li>
-           <a href="mailto:?subject=I love owls :: codebar">Email a friend</a>
+           <a href="mailto:?subject=I love owls :: Codebar">Email a friend</a>
```
Comment (KimberleyCook) [inline html/lesson2/example.html]: Same here, lowercase c

File: html/lesson2/example.html (hunk 1 above)
Comment (stampycode) [reply html/lesson2/example.html]: Ah, no worries - there was an uppercase C here:
http://tutorials.codebar.io/html/lesson2/tutorial.html

The text
> Download the files required to begin working through the tutorial from here (mac/linux)

Links to a zip file here:
`https://gist.github.com/hundred/7332441/download`

I didn't know which was correct. 👍

Comment (stampycode) [reply html/lesson2/example.html]: (I'll update the PR tomorrow.)

Comment (KimberleyCook) [reply html/lesson2/example.html]: thanks :)

**Comment by koddsson** [comment]
This looks good to me! ✨

**Comment by despo** [comment]
Thank you for the PR @stampycode and @koddsson for taking the time to review! :shipit:

**Pull request merged**